installation/sql/mysql/base.sql

```diff
@@ -1168,3 +1168,6 @@ INSERT INTO `#__workflow_transitions` (`id`, `asset_id`, `published`, `ordering`
 (5, 62, 1, 5, 1, 'Feature', '', -1, 1, '{"featuring":"1"}'),
 (6, 63, 1, 6, 1, 'Unfeature', '', -1, 1, '{"featuring":"0"}'),
 (7, 64, 1, 7, 1, 'Publish & Feature', '', -1, 1, '{"publishing":"1","featuring":"1"}');
```

Comment:
This is wrong. I said you have to add id to the other insert statement for extensions here:

https://github.com/joomla-projects/gsoc21_cookie-manager/blob/e0d5a7b8dcec2e1603aade907a632ba5d104f24e/installation/sql/mysql/base.sql#L182

Reply:
means have to add sql at line no.183

Reply:
yes, appended with a comma, and the semicolon at the end then.

same for the postgresql script.

Reply:
and using the same columns, i.e. it has the id in it, too.

but i have explained all that in a previous comment, so maybe check again.

Reply:
ah and i see there is no hardcoded id, my memory was wrong.

Reply:
So it should look like this at the end with your statement appended:
```
(0, 'com_workflow', 'component', 'com_workflow', '', 1, 1, 0, 1, 1, '', '{}', ''),
(0, 'com_mails', 'component', 'com_mails', '', 1, 1, 1, 1, 1, '', '', ''),
(0, 'com_cookiemanager', 'component', 'com_cookiemanager', '', 1, 1, 1, 0, 1, '', '{"policylink":"","position":"bottom"}', '');
```

Reply:
I am also searching for id 😀